node/nodeTransactions.go

```diff
@@ -0,0 +1,135 @@
+package node
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+)
+
+// GetTransaction gets the transaction
+func (n *Node) GetTransaction(txHash string) (*transaction.Transaction, error) {
+	if !n.apiTransactionByHashThrottler.CanProcess() {
+		return nil, ErrSystemBusyTxHash
+	}
+
+	n.apiTransactionByHashThrottler.StartProcessing()
+	defer n.apiTransactionByHashThrottler.EndProcessing()
+
+	hash, err := hex.DecodeString(txHash)
+	if err != nil {
+		return nil, err
+	}
+
+	txBytes, found := n.getTxFromDataPool(hash)
```

Comment:
this name of method is confusing getTxFromDataPool because i m not getting a transaction i m getting a slice of bytes....

Reply:
renamed to `getTxBytesFromDataPool`